src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -172,9 +172,15 @@
         content: ': ';
     }
 
-    &.required {
+    &.required:not(.ez-data-source__label) {
```

Comment:
I think we can avoid adding `:not(.ez-data-source__label)` here, as the next selector is a stronger than this one.